sympy/concrete/summations.py

```diff
@@ -1137,9 +1137,16 @@ def eval_sum_symbolic(f, limits):
         if without_i != 0:
             s = eval_sum_symbolic(with_i, (i, a, b))
             if s:
-                r = without_i*(b - a + 1) + s
-                if r is not S.NaN:
-                    return r
+                if((s == S.NegativeInfinity and (b - a + 1) == S.Infinity) or (s == S.Infinity and (b - a + 1) == S.Infinity)):
+                    Max = max(abs(f.subs(i, a)), abs(f.subs(i, S.Infinity)))
+                    if(Max > abs(without_i)):
```

Comment:
No brackets around the comparison. You may also want to consider that this may not evaluate and then you will get an error. Better to use `is_gt` and then check the three cases: `True`, `False` and `None`. (Or maybe you have some indication from previous code flow that it will evaluate?)

Reply:
ahh yeah the general case where the limits are of the form `(x , int, oo)` is handled properly by `eval_sum_hyper() `method ,no need to worry about that . 
```
    # Now b == oo, a != -oo
    res = _eval_sum_hyper(f, i, a)
    if res is not None:
        r, c = res
        if c == False:
            if r.is_number:
                f = f.subs(i, Dummy('i', integer=True, positive=True) + a)
                if f.is_positive or f.is_zero:
                    return S.Infinity
                elif f.is_negative:
                    return S.NegativeInfinity
            return None
        return Piecewise(res, (old_sum, True))
```
To be honest this actually boils down to a very very specific case that i've tried to fix . Sympy wasn't able to deal with any sort of numbers  or non integer exponents contained in Sum but as obviously as integers and real are infinite in number even this specific case becomes very important to fix .I've shown above what all would be fixed as an end result ,and all of these weren't responding as expected .